### PR TITLE
CCv0: Use cached kernel

### LINKED ISSUE #5355: CC | Use cached build for the cc-kernel-tarball when possible
One way to speed up the current CC CIs is by caching the artefacts that don't change frequently and simply download and use them whenever they're needed.

This issue is for using the cached tarball whenever it's possible, instead of building the component.

### PR DESCRIPTION
**Pull request by GabyCT**
This PR enables the use for the cached kernel.

Fixes #5355

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: tools/packaging/static-build/kernel/build.sh [hunk 1]
```diff
@@ -17,25 +17,68 @@ source "${script_dir}/../../scripts/lib.sh"
...
+	mkdir -p "${kernel_directory}"
+	pushd "${kernel_directory}"
+	curl -fL --progress-bar "${kernel_latest_build_url}/${kernel_tarball_name}" -o "${kernel_tarball_name}" || return 1
+	curl -fsOL "${kernel_latest_build_url}/sha256sum-${kernel_tarball_name}" || return 1
+	sha256sum -c "sha256sum-${kernel_tarball_name}" || return 1
+	popd
+
+}
+
+calc_kernel_files_sha256sum() {
```
[the remark this answers is not in the document]
Comment (GabyCT) [reply tools/packaging/static-build/kernel/build.sh:72]: thanks @wainersm

**Comment by GabyCT** [comment]
@wainersm and @fidencio could you please take a look? thanks

**Review by fidencio — APPROVED** [review]
lgtm, thanks @GabyCT!

**Pull request merged**